documentation/how-to/prepare-for-production.md

```diff
@@ -0,0 +1,193 @@
+
+# Prepare for Production
+
+## Performance Optimisation
+
+### Enable Asynchronous Querying
```

Comment:
```suggestion
### Enable asynchronous querying
```
We go sentence case for titles. This applies to all cases. I link you here the Canonical style guide: https://docs.ubuntu.com/styleguide/en?_ga=2.205521435.902454895.1715672178-1762339271.1714993055#headings

Reply:
Updated across all titles, thanks!